Mathlib/CategoryTheory/Preadditive/Biproducts.lean

```diff
@@ -151,7 +151,6 @@ def biconeIsBilimitOfLimitConeOfIsLimit {f : J → C} {t : Cone (Discrete.functo
     IsLimit.ofIsoLimit ht <|
       Cones.ext (Iso.refl _)
         (by
-          rintro ⟨j⟩
           aesop_cat)
```

Comment:
Can you write this on one line now, as `(by aesop_cat)`?

Reply:
done, I formatted it to fit everything on one line